src/applications/_mock-form/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import moment from 'moment';
```

Comment:
Can we use [date-fns](https://date-fns.org/) here instead of moment? We're trying to gradually get it out of the system because it's massive and deprecated